annotations_to_biigle/annotations_to_biigle.py

```diff
@@ -10,7 +10,7 @@
 
 # Biigle email and token
 email = 'mmarcill@ifremer.fr'
-token = 'jxSmYAj3uM7OOA1GIp5w2wCOyYMhb7vo'
```

Comment:
Oops. Don't forget to revoke the token in BIIGLE. I'll review the PR soon!

Reply:
Oops, thanks ! Done